pandas/tests/plotting/test_backend.py

```diff
@@ -36,3 +42,44 @@ def test_backend_is_correct(monkeypatch):
         pandas.set_option("plotting.backend", "matplotlib")
     except ImportError:
         pass
+
+
+@td.skip_if_no_mpl
+def test_register_entrypoint():
+    mod = types.ModuleType("my_backend")
+    mod.plot = lambda *args, **kwargs: 1
+
+    backends = pkg_resources.get_entry_map("pandas")
+    my_entrypoint = pkg_resources.EntryPoint(
+        "pandas_plotting_backend",
+        mod.__name__,
+        dist=pkg_resources.get_distribution("pandas"),
+    )
+    backends["pandas_plotting_backends"]["my_backend"] = my_entrypoint
+    # TODO: the docs recommend importlib.util.module_from_spec. But this works for now.
+    sys.modules["my_backend"] = mod
+
+    result = pandas.plotting._core._get_plot_backend("my_backend")
+    assert result is mod
+
+
+@td.skip_if_no_mpl
+def test_register_import():
+    mod = types.ModuleType("my_backend2")
+    mod.plot = lambda *args, **kwargs: 1
+    sys.modules["my_backend2"] = mod
+
+    result = pandas.plotting._core._get_plot_backend("my_backend2")
+    assert result is mod
+
+
+def test_no_matplotlib_ok():
+    try:
+        import matplotlib  # noqa
+    except ImportError:
+        pass
+    else:
+        raise pytest.skip("matplotlib installed.")
```

Comment:
Don't we have a decorator `skip_if_mpl`? May be I'm missing something, but seems like should be the same.